docs/ADR/Eksponere endepunkt for beregning av totalsum.md

```diff
@@ -0,0 +1,23 @@
+# Eksponere endepunkt for beregning av totalsum
```

Comment:
Å begynne filnavn med et nummer kan være nyttig for sortering av ADR-er

Reply:
Refaktorert